client/kv.go

```diff
@@ -1,13 +1,54 @@
 package client
 
 import (
-	"errors"
 	"fmt"
 
 	"github.com/google/uuid"
 	xlineapi "github.com/xline-kv/go-xline/api/xline"
 )
 
+type KV interface {
+	// Get retrieves keys.
+	// By default, Get will return the value for "key", if any.
+	// When passed WithRange(end), Get will return the keys in the range [key, end).
+	// When passed WithFromKey(), Get returns keys greater than or equal to key.
+	// When passed WithRev(rev) with rev > 0, Get retrieves keys at the given revision;
+	// if the required revision is compacted, the request will fail with ErrCompacted .
+	// When passed WithLimit(limit), the number of returned keys is bounded by limit.
+	// When passed WithSort(), the keys will be sorted.
+	Get(key string, opts ...OpOption) (*GetResponse, error)
+
+	// Put puts a key-value pair into etcd.
+	// Note that key,value can be plain bytes array and string is
+	// an immutable representation of that bytes array.
+	// To get a string of bytes, do string([]byte{0x10, 0x20}).
+	Put(key, val string, opts ...OpOption) (*PutResponse, error)
```

Comment:
Why should we specify the type of the value?  Can we use general type here?

Reply:
To easy operation of simple data. We can use a more general type `[]byte`.